lib/lambda-cfn.js

```diff
@@ -24,6 +24,28 @@ lambdaCfn.lambdaSnsUser = lambdaSnsUser;
 lambdaCfn.lambdaSnsUserAccessKey = lambdaSnsUserAccessKey;
 lambdaCfn.outputs = outputs;
 lambdaCfn.load = load;
+lambdaCfn.getEnv = getEnv;
+
+function stripHyphens(r) {
+    return r.replace(/-/g, '');
+}
+
+var packages;
+if (process.env.NODE_ENV == 'test') {
+  packages = require(path.join(root,'/test/package.json'));
+} else {
+  packages = require(path.join(root,'package.json'));
+}
+
+function namespace(name,path) {
```

Comment:
@ianshward thanks for pointing that out, looking at that object, I think I can crawl it to extract the right information and avoid this ugly hacking about with the package.json.